.github/workflows/build-windows.yml

```
@@ -200,7 +200,7 @@ jobs:
       # so pass --ignore-dep ninja explicitly
       - name: Build and install pygame wheel
         shell: msys2 {0}
-        run: python3 dev.py --ignore-dep ninja build --wheel /artifacts --lax
+        run: python3 dev.py --ignore-dep ninja build --wheel /artifacts --lax  --ci --commit_hash ${{ github.sha }} --branch "${{ github.head_ref || github.ref_name }}"
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)